workbench-core/example/infrastructure/package.json

```diff
@@ -31,7 +31,7 @@
     "pkg-json-lint": "npmPkgJsonLint -c ../../../.npmpackagejsonlintrc.json .",
     "sort-package-json": "sort-package-json package.json",
     "test": "rushx test:only",
-    "test-cdk-deploy": "rush build && rushx compile && cdk deploy --all --require-approval never --outputs-file src/config/testEnv.json",
+    "test-cdk-deploy": "rush build -t @aws/workbench-core-example-infrastructure && rushx compile && cdk deploy --all --require-approval never --outputs-file src/config/testEnv.json",
```

Comment:
will this ensure that ALL changes are picked up and packaged into the lambda handler? Or will it just rebuild the infrastructure package and potentially miss some things?

Reply:
This builds the example infrastructure package and all dependencies (and dependencies of dependencies)